packages/colorpickers/demo/colorPickerDialog.stories.mdx

```diff
@@ -7,7 +7,11 @@ import { ColorpickerDialogStory } from './stories/ColorpickerDialogStory';
 <Meta
   title="Packages/Colorpickers/ColorpickerDialog"
   component={ColorpickerDialog}
-  args={{ buttonProps: { 'aria-label': 'Label' }, isAnimated: true }}
+  args={{
+    buttonProps: { 'aria-label': 'Label' },
+    'aria-label': 'Storybook color picker',
```

Comment:
[nit] recommend simplifying to "Title" to be consistent with other usage and more explicitly identify what the control is functioning as.